.reek

```diff
@@ -0,0 +1,21 @@
+Attribute:
+  enabled: false
+IrresponsibleModule:
+  enabled: false
+TooManyStatements:
+  max_statements: 6
+UtilityFunction:
+  public_methods_only: true
+'spec':
+  FeatureEnvy:
+    enabled: false
+  TooManyInstanceVariables:
+    enabled: false
+  TooManyMethods:
+    enabled: false
+  TooManyStatements:
+    enabled: false
+  UtilityFunction:
+    enabled: false
+exclude_paths:
+  - db/migrate
```

Comment:
glad we are adding rubocop and reek!!! Can we do this in a separate PR, though? Feels unrelated to this exact change